app/services/participants/ecf.rb

```diff
@@ -6,7 +6,7 @@ module ECF
 
     included do
       delegate :early_career_teacher?, :mentor_profile, :early_career_teacher_profile, :participant?, to: :user, allow_nil: true
-      delegate :school_cohort, :participant_profile_state, to: :user_profile, allow_nil: true
+      delegate :school_cohort, :participant_profile_state, :validate_participant_state, to: :user_profile, allow_nil: true
```

Comment:
Don't think we need this in participant services.

Reply:
Left there after refactor, will remove